spec/support/http_library_adapters.rb

```diff
@@ -227,7 +227,11 @@ def make_http_request(method, url, body = nil, headers = {})
   end
 
   def normalize_request_headers(headers)
-    headers.merge('User-Agent' => [Excon::USER_AGENT])
+    if Excon::VERSION.start_with?("0.63")
```

Comment:
```suggestion
    if Gem::Version.new(Excon::VERSION) < Gem::Version.new("0.63")
```

Thanks @terceiro for creating this thingie, perhaps it could help here?

Reply:
I'm testing this too. As soon as I have the travis results I'll get back to you.

Reply:
Argh. The change in <https://github.com/vcr/vcr/pull/809/commits/da0714182b260022b7bb185ce8bac53d827134c5> changed the test logic and therefor the Accept header got added for versions lower 0.63. @terceiro fixed the test logic again.